services/workflows-service/src/ui-definition/ui-definition.service.ts

```diff
@@ -102,4 +105,42 @@ export class UiDefinitionService {
 
     return uiDefinitionCopy;
   }
+
+  traverseUiSchema(
```

Comment:
Moved from the collection flow service to a shared service that makes sense considering functionality.